.eslintrc.json

```diff
@@ -19,111 +18,186 @@
     "parser": "@typescript-eslint/parser",
```

Comment:
btw eslint is deprecating
https://eslint.org/blog/2023/10/deprecating-formatting-rules/